### PR TITLE
earntype,employeecategory,grademaster,earnings/deduction

### PR DESCRIPTION
**Pull request by ReshmaRaj1994**

### CODE REVIEW

File: src/views/Master/EarningDeductionMaster/EarnDectEdit.jsx [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+    // on change function 
+    const updateFormData = (e) => {
+        const value = e.target.type === 'checkbox' ? e.target.checked : e.target.value;
+
+        setearnDeduct({ ...earndecData, [e.target.name]: value })
+    }
+
+    //default state
+    const deful = {
```
Comment (itmedicity) [inline src/views/Master/EarningDeductionMaster/EarnDectEdit.jsx:77]: unidentified Name

File: src/views/Master/EmploymentType/EmploymentTypetable.jsx [hunk 2]
```diff
@@ -48,8 +48,8 @@ const EmploymentTypetable = () => {
     }
 
     const getTablerowData = (data) => {
-        const { emt_slno } = data;
```
Comment (itmedicity) [inline src/views/Master/EmploymentType/EmploymentTypetable.jsx:51]: Repeated Destructuring

File: src/views/Master/GradeMaster/GradeMastEdit.jsx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    const postData = {
+
+        grade_desc: gradename,
+        grade_status: grade_status === true ? 1 : 0,
+        grade_slno: id,
+    }
+
+    const postFormData = async (e) => {
+        e.preventDefault();
+
```
Comment (itmedicity) [inline src/views/Master/GradeMaster/GradeMastEdit.jsx:75]: Remove the Unwanted Space

**Review by itmedicity — CHANGES_REQUESTED** [review]

File: src/views/Master/EmploymentType/EmploymentTypetable.jsx (hunk 2 above)
Comment (ReshmaRaj1994) [reply src/views/Master/EmploymentType/EmploymentTypetable.jsx:51]: ok

File: src/views/Master/GradeMaster/GradeMastEdit.jsx (hunk 3 above)
Comment (ReshmaRaj1994) [reply src/views/Master/GradeMaster/GradeMastEdit.jsx]: ok

File: src/views/Master/EarningDeductionMaster/EarnDectEdit.jsx (hunk 1 above)
Comment (ReshmaRaj1994) [reply src/views/Master/EarningDeductionMaster/EarnDectEdit.jsx:77]: ok

**Pull request closed without merging**